### PR TITLE
Longer security considerations section

### PR DESCRIPTION
**Pull request by rohan-wire**

### CODE REVIEW

File: draft-ralston-mimi-protocol.md [hunk 1]
```diff
@@ -1499,10 +1499,26 @@ be sent by an authorized external sender.
...
+verified by the end clients involved in the room.  The usage of MLS ensures that
+the servers facilitating the exchange cannot read messages in the room or
+falsify room state changes, even though they can read the room state change
+messages.
+
+Each room has an authorization policy that dictates which protocol actors can
+perform which actions in the room.  This policy is enforced by the hub server
+for the room.  The actors for whom the policy is being evaluated authenticate
+their identities to the hub server using the MLS PublicMessage signed object
+format, together with the identity credentials presented in MLS.
```
Comment (turt2live) [inline draft-ralston-mimi-protocol.md:1521]: we may need to add a note that the hub-and-spoke model allows the hub to be a threat actor itself, but is deemed appropriate for simplicity.

**Review by turt2live — COMMENTED** [review]
otherwise lgtm

File: draft-ralston-mimi-protocol.md (hunk 1 above)
Comment (rohan-wire) [reply draft-ralston-mimi-protocol.md:1521]: good point

Comment (bifurcation) [reply draft-ralston-mimi-protocol.md:1521]:
```suggestion
Each room has an authorization policy that dictates which protocol actors can
perform which actions in the room.  This policy is enforced by the hub server
for the room.  The actors for whom the policy is being evaluated authenticate
their identities to the hub server using the MLS PublicMessage signed object
format, together with the identity credentials presented in MLS.  This design
means that the hub is trusted to correctly enforce the room's policy, but this 
cost is offset by the simplicity of not having multiple policy enforcement points.
```

**Review by bifurcation — APPROVED** [review]

**Pull request merged**